XamlControlsGallery/ControlPages/NavigationViewPage.xaml

```diff
@@ -12,13 +12,16 @@
     <Page.Resources>
         <converters:MenuItemTemplateSelector x:Key="selector" >
             <converters:MenuItemTemplateSelector.ItemTemplate>
-                <DataTemplate x:DataType="converters:Category">
+                <DataTemplate x:DataType="converters:CategoryWithBadgeValue">
                     <muxc:NavigationViewItem 
                                 Content="{x:Bind Name}"
                                 ToolTipService.ToolTip="{x:Bind Tooltip}">
                         <muxc:NavigationViewItem.Icon>
                             <SymbolIcon Symbol="{x:Bind Glyph}" />
                         </muxc:NavigationViewItem.Icon>
+                        <muxc:NavigationViewItem.InfoBadge>
+                            <muxc:InfoBadge Value="{x:Bind Value}"/>
+                        </muxc:NavigationViewItem.InfoBadge>
```

Comment:
Two things here:
1. Those badges only seem to show on hover instead of being immediatly visible when scrolling to the sample, maybe this is a WinUI issue?
2. You will also need to udpate the template code for this. You can find the sample code here: https://github.com/microsoft/Xaml-Controls-Gallery/blob/master/XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample5_xaml.txt